build/escargot.cmake

```diff
@@ -111,27 +117,27 @@ IF (${ESCARGOT_OUTPUT} STREQUAL "bin")
     ADD_EXECUTABLE (${ESCARGOT_TARGET} ${ESCARGOT_SRC_LIST})
     ADD_DEPENDENCIES (${ESCARGOT_TARGET} gc)
 
-    TARGET_LINK_LIBRARIES (${ESCARGOT_TARGET} ${ESCARGOT_LIBRARIES} ${GC_TARGET} ${ESCARGOT_LDFLAGS} ${LDFLAGS_FROM_ENV})
+    TARGET_LINK_LIBRARIES (${ESCARGOT_TARGET} ${ESCARGOT_LIBRARIES} ${GC_TARGET} ${ESCARGOT_LDFLAGS} ${ESCARGOT_LDFLAGS_ASAN} ${LDFLAGS_FROM_ENV})
     TARGET_INCLUDE_DIRECTORIES (${ESCARGOT_TARGET} PUBLIC ${ESCARGOT_INCDIRS})
     TARGET_COMPILE_DEFINITIONS (${ESCARGOT_TARGET} PUBLIC ${ESCARGOT_DEFINITIONS})
-    TARGET_COMPILE_OPTIONS (${ESCARGOT_TARGET} PUBLIC ${ESCARGOT_CXXFLAGS} ${CXXFLAGS_FROM_ENV} ${PROFILER_FLAGS})
+    TARGET_COMPILE_OPTIONS (${ESCARGOT_TARGET} PUBLIC ${ESCARGOT_CXXFLAGS} ${ESCARGOT_CXXFLAGS_ASAN} ${CXXFLAGS_FROM_ENV} ${PROFILER_FLAGS})
```

Comment:
Instead of adding cmake variables into each target, how about adding these flags into the existing variables like below?
`SET (ESCARGOT_CXXFLAGS ${ESCARGOT_CXXFLAGS} -fsanitize=address)`
or
`SET (ESCARGOT_CXXFLAGS ${ESCARGOT_CXXFLAGS} ${ESCARGOT_CXXFLAGS_ASAN})`

Reply:
I removed it